### DISCUSSION TITLE
Question about Utilities / Export to Disk

### DISCUSSION THREAD

**Discussion by bkrepp-ctps** [requarks/wiki#5977] [Help / Questions] [answered]
Hi al,
This question may be related to my previous one about 'Backups and Dumps', but since it involves use of a different part of the wiki.js administrative UI, I thought it best to ask it separately. Here goes.

First, I create a folder into which the export is to be written: /tmp/export_12202022, and set its permission to 755,
Then, under 'Admin' / 'Utilities' / 'Export to Disk', I do the following
- Select various assets to export
- Enter the **full, absolute path** as the 'Target Folder Path', i.e., /tmp/export_12202022
- Press the 'Start Export' button

The export appears to run, and after a short while a message is displayed reporting that the export is complete.

Yet, when I look at the contents of /tmp/export_12202022, I find that it is **empty**.

Can someone clarify what is going on? In particular:
- Has the export actually been created, and if so am I simply looking in the wrong place for it?
- Has the export not been created (in spite of the 'export completed' message, and if so what did I do incorrectly?

Thanks for any help you can offer with these questions.
-- Ben Krepp

**Comment by NGPixel** [comment, marked as answer]
Are you running in Docker? If so, the path will be inside the container, not on your host.

**Reply by bkrepp-ctps** [reply]
Nicolas, 
First, thanks very much taking the time to reply to my question. 
It's a bit difficult to answer _your_ question because I'm a bit new to the "world" in which my wiki is running, and not 100% sure of what's going on "under the hood".  Here's what I know:

1. I created my wiki by launching a Digital Ocean 'Droplet' that was pre-packaged (not sure if that's the correct term) with wiki.js.
2. Once the 'Droplet' was launched, I was given its IP address, and with that I was able to connect to the wiki and start entering content.
3. 
I _think_ that my wiki is running in a Docker container in the DO 'Droplet', but that's not obvious (at least not to me), 'from the outside', i.e., by looking at my Digital Ocean console. 

If I log into my Droplet however, and run the Linux 'ps' command, I see that there are processes running with names like /usr/bin/dockerd and  /user/bin/docker-proxy. Given this, I _think_ my wiki is running inside one of those Docker processes, but don't feel 100% confident saying so, because I don't know how to look 'inside' those processes to see what's running inside of them.

Given the above info, would you conclude that my wiki is running under one of those Docker processes?
If so, how do I find which one, and again, how would I find my way to where the exported data can actually be found.

As must be obvious from my questions, I'm new to the Docker world. 
Please pardon my 'newbie' questions, and thank you for any further help you can offer with them.
All best,
Ben

**Reply by NGPixel** [reply]
The prebuilt wiki.js droplet is using docker. You can see running containers using command `docker ps`. You'll notice 3 running containers `db`, `wiki` and `wiki-update-companion`.

You can copy a file or folder from the wiki docker container to the host using command `docker cp`, e.g.:

```sh
docker cp wiki:/tmp/export_12202022/ /tmp/export_12202022
```

**Comment by bkrepp-ctps** [comment]
Many thanks for your reply, Nicolas. I've successfully copied my export directory to the Droplet's file system.
